src/_P039_Thermosensors.ino

```diff
@@ -737,11 +763,15 @@ float readMax6675(struct EventStruct *event)
       String log;
       if((log.reserve(70u))) {
         log = F("P039 : MAX6675 : RAW - BIN:");     // 27 char
-        log += String(rawvalue, BIN);               // 18 char
-        log += F(" HEX:");                          // 5 char
-        log += String(rawvalue, HEX);               // 4 char
+        log += String(rawvalue, BIN);          // 18 char
+        log += F(" HEX: 0x");                          // 5 char
+        log += String(rawvalue, HEX);          // 4 char
         log += F(" DEC:");                          // 5 char
-        log += String(rawvalue);                    // 5 char
+        log += String(rawvalue);               // 5 char
+        log += F(" MSB: 0x");                          // 5 char
+        log += String(messageBuffer[0], HEX);          // 5 char 
+        log += F(" LSB: 0x");                          // 5 char
+        log += String(messageBuffer[1], HEX);          // 5 char
```

Comment:
What I usually do, is check the actual logs and use that to estimate the expected size of the string.
Computing it using comments is a lot of work and really hard to maintain.

Reply:
True - this is work for the black sheep of the family. I follow your proposal...

Reply:
This will require another two or three days...

Reply:
Solved in latest push,